crates/ruff_linter/src/test.rs

```diff
@@ -239,15 +239,15 @@ Source with applied fixes:
             let fixable = diagnostic.fix.is_some();
 
             match (fixable, rule.fixable()) {
-                (true, FixKind::Sometimes | FixKind::Always)
-                | (false, FixKind::None | FixKind::Sometimes) => {
+                (true, FixAvailability::Sometimes | FixAvailability::Always)
+                | (false, FixAvailability::None | FixAvailability::Sometimes) => {
                     // Ok
                 }
-                (true, FixKind::None) => {
-                    panic!("Rule {rule:?} is marked as non-fixable but it created a fix. Change the `Violation::FIX_KIND` to either `FixKind::Sometimes` or `FixKind::Always`");
+                (true, FixAvailability::None) => {
+                    panic!("Rule {rule:?} is marked as non-fixable but it created a fix. Change the `Violation::FIX_AVAILABILITY` to either `FixAvailability::Sometimes` or `FixAvailability::Always`");
                 },
-                (false, FixKind::Always) => {
-                    panic!("Rule {rule:?} is marked to always-fixable but the diagnostic has no fix. Either ensure you always emit a fix or change `Violation::FIX_KINDd` to either `FixKind::Sometimes` or `FixKind::None")
+                (false, FixAvailability::Always) => {
+                    panic!("Rule {rule:?} is marked to always-fixable but the diagnostic has no fix. Either ensure you always emit a fix or change `Violation::FIX_AVAILIBITYd` to either `FixAvailability::Sometimes` or `FixAvailability::None")
```

Comment:
Looks like a typo in `Violation::FIX_AVAILIBITYd` (which was already present before, but worth fixing).